code/modules/clothing/head/helmets/misc.dm

```diff
@@ -429,8 +429,9 @@
 	smeltresult = /obj/item/ingot/iron
 	sellprice = VALUE_IRON_ARMOR_UNUSUAL
 	item_weight = 6 * IRON_MULTIPLIER
+	detail_tag = "_detail"
 	detail_color = CLOTHING_PLUM_PURPLE
-	uses_lord_coloring = LORD_PRIMARY
```

Comment:
shouldn't use regular color it looks like ass, just LORD_PRIM/SEC is detail if its present

Reply:
because tabards are greyscale so they don't look like ass, if it doesn't work like every other lord color clothing item which I wrote the for you have done something wrong

Reply:
they aren't the same as tabbards because tabbards have no detail so `LORD_PRIMARY` changes `color` to `GLOB.lord_primary`
Your thing **HAS** detail so LORD_PRIMARY changes `detail_color` to `GLOB.lord_primary` if you then add `LORD_DETAIL_AND_COLOR` `color` also changes to `GLOB.lord_primary` which looks bad.

Reply:
my auto doc for that is a bit misleading at best

Reply:
and fix it without changing its typing